test/integration/connect/envoy/case-expose-checks/verify.bats

```diff
@@ -37,5 +37,6 @@ load helpers
 }
 
 @test "s2 exposes checks on a new listener" {
+exit 1
```

Comment:
This fails the test, right? Is that intended?

Reply:
Ha. I did not intend to commit it. Removed.

I did this because it triggers the `capture.sh` script to run when the test fails so all of the envoy information is written to the `workdir` along with the logs so I could figure out what was happening. I just forgot to remove it.